VideoApi/VideoApi/Mappings/ConferenceForHostResponseMapper.cs

```diff
@@ -25,14 +25,31 @@ public static ConferenceForHostResponse MapConferenceSummaryToModel(Conference c
                 NumberOfEndpoints = conference.Endpoints.Count
             };
         }
-        
-        public static IEnumerable<JudgeInHearingResponse> MapConferenceSummaryToJudgeInHearingResponse(Conference conference)
+
+        public static IEnumerable<ParticipantInHearingResponse> MapConferenceSummaryToJudgeInHearingResponse(Conference conference)
+        {
+            var conferenceId = conference.Id;
+
+            return conference.Participants
+            .Where(x => x is Participant && ((Participant)x).IsJudge())
+            .Select(x => new ParticipantInHearingResponse
+            {
+                Id = x.Id,
+                ConferenceId = conferenceId,
+                Status = x.State.MapToContractEnum(),
+                Username = x.Username,
+                CaseGroup = ((Participant)x).CaseTypeGroup,
+                UserRole = x.UserRole.MapToContractEnum()
+            });
+        }
+
+        public static IEnumerable<ParticipantInHearingResponse> MapConferenceSummaryToHostInHearingResponse(Conference conference)
```

Comment:
Again just to reduce the duplication could the logic be moved into a private method that takes a list of roles to match by that these 2 could call?